drivers/node/ssh/ssh.go

```diff
@@ -731,7 +731,7 @@ func (s *SSH) SystemCheck(n node.Node, options node.ConnectionOpts) (string, err
 // GetBlockDrives returns the block drives on the node
 func (s *SSH) GetBlockDrives(n node.Node, options node.SystemctlOpts) (map[string]*node.BlockDrive, error) {
 	drives := make(map[string]*node.BlockDrive)
-	driveCmd := fmt.Sprintf("sudo /bin/lsblk -P -s -d -p -o NAME,SIZE,MOUNTPOINT,FSTYPE,TYPE")
+	driveCmd := fmt.Sprintf("sudo /bin/lsblk --fs -P -s -d -p -o NAME,LABEL,SIZE,MOUNTPOINT,FSTYPE,TYPE")
```

Comment:
instead of "--fs" can we use "-f" ?

Reply:
-f, --fs
           Output info about filesystems.

Reply:
fixed